libnopegl/src/api.c

```diff
@@ -196,8 +196,6 @@ void ngli_ctx_reset(struct ngl_ctx *s, int action)
 #if defined(TARGET_ANDROID)
     ngli_android_ctx_reset(&s->android_ctx);
 #endif
-    ngli_atlas_freep(&s->font_atlas); // allocated by the first node text
```

Comment:
typo in commit desc: futur -> future.

Reply:
Fixed, and changed with your suggestion (just removed the comma)